### DISCUSSION TITLE
Help.

### DISCUSSION THREAD

**Discussion by fard33n** [qnblackcat/uYouPlus#157] [Q&A]
Is there a way to remove the uYou button from the bottom of the app?

![20220513_210350.jpg](https://user-images.githubusercontent.com/105258592/168317811-d83772fb-5ad3-4b70-9b2e-29a5d2a85c71.jpg)

**Comment by qnblackcat** [comment]
You should ask MiRO92, the dev of uYou

**Comment by Skywalker8510** [comment]
i dont believe there is a way to do that because you have to enter that tab to change any uYou settings
